backend/ddang/src/main/java/com/ddang/ddang/auction/application/AuctionService.java

```diff
@@ -89,17 +96,19 @@ private List<AuctionImage> convertAuctionImages(final CreateAuctionDto dto) {
                              .toList();
     }
 
-    public ReadAuctionDto readByAuctionId(final Long auctionId) {
-        final Auction auction = findAuction(auctionId);
-
-        return ReadAuctionDto.from(auction);
-    }
-
-    private Auction findAuction(final Long auctionId) {
-        return auctionRepository.findAuctionById(auctionId)
-                                .orElseThrow(() -> new AuctionNotFoundException(
-                                        "지정한 아이디에 대한 경매를 찾을 수 없습니다."
-                                ));
+    public ReadAuctionWithChatRoomIdDto readByAuctionId(final Long auctionId, final AuthenticationUserInfo userInfo) {
+        final Auction findAuction = auctionRepository.findAuctionById(auctionId)
+                                                     .orElseThrow(() -> new AuctionNotFoundException(
+                                                             "지정한 아이디에 대한 경매를 찾을 수 없습니다."
+                                                     ));
+        final User findUser = userRepository.findById(userInfo.userId())
+                                            .orElseThrow(() -> new UserNotFoundException("회원 정보를 찾을 수 없습니다."));
+        final Optional<ChatRoom> nullableChatRoom = chatRoomRepository.findByAuctionId(findAuction.getId());
```

Comment:
```suggestion
        final Long chatRoomId = chatRoomRepository.findByAuctionId(findAuction.getId())
                                          .map(auction -> auction.getId())
                                          .orElse(null);
```

Optional을 매개변수로 전달하는 것은 최대한 지양하고 Optional에서 제공해주는 메서드로 Auction을 id로 변환하는 것이 좋을 것 같습니다

Reply:
저 위에서 null을 상수로 DEFAULT_CHAT_ROOM_ID나 NOT_EXISTS_CHAT_ROOM_ID같은걸로 변경하면 조금 더 의미를 부여해줄 수 있을 것 같습니다

Reply:
왜 저는 이런 생각이 안날까요..😢
코드 제안 감사합니다 지토!